.github/workflows/main.yml

```diff
@@ -1,24 +1,23 @@
-name: Main Workflow
-on: [push]
-jobs:
-  build:
-    name: Build
-    strategy:
-      matrix:
-        rust-version: ["1.40", "1.45", "1.46", "1.47", "1.48", "1.49", latest]
-    runs-on: ubuntu-latest
-    container: rust:${{ matrix.rust-version }}
-    steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-python@v2
-        with:
-          python-version: "3.8"
-      - run: cargo build
-      - run: cargo test
-      - run: cargo build --features python-extension
-      - run: cargo build --features wasm-extension
-      - run: cargo run version
-      - run: cargo run benchmark assets/demo/
-      - run: cargo run benchmark assets/demo/ --parallel
-      - run: cargo doc --lib --all-features
-      - run: ${pythonLocation}/bin/python3 -m pip install --upgrade .
+name: Main Workflow
+on: [push]
+jobs:
+  build:
+    name: Build
+    strategy:
+      matrix:
+        rust-version: ["1.40", "1.45", "1.46", "1.47", "1.48", "1.49", latest]
+    runs-on: ubuntu-latest
+    container: rust:${{ matrix.rust-version }}
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - run: cargo build
+      - run: cargo test
+      - run: cargo run version
+      - run: cargo run benchmark assets/demo/
+      - run: cargo run benchmark assets/demo/ --parallel
+      - run: cargo doc --lib --all-features
+      - run: ${pythonLocation}/bin/python3 -m pip install --upgrade .
+      - run: ${pythonLocation}/bin/python3 setup.py test
```

Comment:
Make a minimal set of changes. If you only want to add 1 line then don't do other things like changing the position of `cargo run version` or how the build commands are called.

Reply:
I didn't change `cargo run version`, I deleted a couple of lines that built the project with WASM and Python extensions. But these build commands are already issued by `pip install` on `main.yaml` and with `wasm-pack build` on `wasm.yaml`